feature-account-impl/src/main/java/io/novafoundation/nova/feature_account_impl/data/repository/addAccount/LocalAddMetaAccountRepository.kt

```diff
@@ -24,6 +22,6 @@ class LocalAddMetaAccountRepository(
     override suspend fun addAccountInternal(payload: Payload): AddAccountResult {
         val metaId = metaAccountDao.insertMetaAccount(payload.metaAccountLocal)
         secretStoreV2.putMetaAccountSecrets(metaId, payload.secrets)
-        return AddAccountResult.AccountAdded(metaId)
+        return AddAccountResult.AccountAdded(metaId, LightMetaAccount.Type.SECRETS)
```

Comment:
We should use type in payload.metaAccountLocal since we can't be sure it's always SECRET